http-clients/src/main/java/com/palantir/remoting3/clients/ClientConfigurations.java

```diff
@@ -51,13 +52,22 @@ private ClientConfigurations() {}
      * empty/absent configuration with the defaults specified as constants in this class.
      */
     public static ClientConfiguration of(ServiceConfiguration config) {
-        return of(config, DEFAULT_RPC_MODE);
+        return of(config, DEFAULT_NODE_SELECTION_STRATEGY);
     }
 
     /**
      * Like {@link #of(ServiceConfiguration)}, but using the explicitly given NodeSelectionStrategy.
      */
     public static ClientConfiguration of(ServiceConfiguration config, NodeSelectionStrategy nodeSelectionStrategy) {
+        return of(config, nodeSelectionStrategy, DEFAULT_FAILED_URL_COOLDOWN);
+    }
+
+    /**
+     * Like {@link #of(ServiceConfiguration, NodeSelectionStrategy)}, with a specified {@link Duration} for the
+     * amount of time URLs marked as failed should be avoided in subsequent calls.
+     */
+    public static ClientConfiguration of(ServiceConfiguration config, NodeSelectionStrategy nodeSelectionStrategy,
```

Comment:
Rather than adding lots of static factories with more and more overridden parameters, can we just recommend people use:

```java
ClientConfiguration.builder()
                .from(ClientConfigurations.of(config))
                . failedUrlCooldown(failedUrlCooldown)
                .build()
```